.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,51 @@
+name: Update
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  update:
+    strategy:
+      matrix:
+        project:
+          - 'scipp/esssans'
+          - 'scipp/sciline'
+          - 'scipp/scippnexus'
+          - 'scipp/scippneutron'
+          - 'scipp/plopp'
+          - 'scipp/essnmx'
+          - 'scipp/essreduce'
+          - 'scipp/beamlime'
+          - 'scipp/essspectroscopy'
+          - 'scipp/chexus'
+          - 'scipp/ess'
+          - 'scipp/chexus'
+          - 'scipp/essreflectometry'
+          - 'scipp/essdiffraction'
+          - 'scipp/esspolarization'
+          - 'scipp/scippuncertainty'
+          - 'scipp/essimaging'
+
+    name: Update downstream projects
+    runs-on: 'ubuntu-latest'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.PAT }}
+          repository: ${{ matrix.project }}
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.X'
+      - run: pip install copier
+      - run: copier update --skip-answered --vcs-ref=HEAD --conflict=rej
```

Comment:
Will it be necessary to run `tox -e deps` here ...?

Reply:
Hmm, I'm not sure. It doesn't hurt to run that too right?